t/01-log_context.t

```diff
@@ -0,0 +1,33 @@
+use Test::More;
+
+use Log::Any::Adapter::DERIV;
+
+our $JSON = JSON::MaybeXS->new(
+    canonical => 1,
+    convert_blessed => 1,
+);
+
+my $adapter = Log::Any::Adapter::DERIV->new;
+my $context = {
+    key1 => 'value1',
+    key2 => 'value2',
+};
+$adapter->set_context($context);
+is_deeply($adapter->{context}, $context, 'set_context sets the context');
+
+my $data = {
+    message  => 'This is a test message "with quotes"',
+    severity => 'info',
+};
+
+foreach my $key (keys %{$context}) {
+    $data->{$key} = $context->{$key};
+}
+
+is($data->{key1}, 'value1', 'key1 is part of data');
+is($data->{key2}, 'value2', 'key2 is part of data');
```

Comment:
I think these are redundant. To test the result of context we need to do something similar to [this](https://github.com/binary-com/perl-Log-Any-Adapter-DERIV/blob/master/t/deriv.t#L51)

Reply:
removed